server/protocols/webadmin-integration-test/memory-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/memory/vault/MemoryLinshareBlobExportMechanismIntegrationTest.java

```diff
@@ -32,8 +31,6 @@
 
 class MemoryLinshareBlobExportMechanismIntegrationTest extends LinshareBlobExportMechanismIntegrationTest {
 
-    private static final LinshareGuiceExtension linshareGuiceExtension = new LinshareGuiceExtension();
-
     @RegisterExtension
     static JamesServerExtension jamesServerExtension = new JamesServerBuilder()
         .extension(linshareGuiceExtension)
```

Comment:
I can't see where does `linshareGuiceExtension` come from. Could you help me?

Reply:
`MemoryLinshareBlobExportMechanismIntegrationTest` should extends  `LinshareGuiceExtension` from `LinshareBlobExportMechanismIntegrationTest`.

Reply:
Ok, spotted. So I disagree with that: the testsuite must provide the tests and the implementation is responsible for setting up the environment.

Factoring things is not always a win if it breaks a design

Reply:
After talked with Benoit, i totally agreed and i will fixed it